whois-nrtm4/src/main/java/net/ripe/db/nrtm4/DeltaChange.java

```diff
@@ -0,0 +1,57 @@
+package net.ripe.db.nrtm4;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+
+
+public class DeltaChange {
+
```

Comment:
I think this is the class we are going to publish? Can you add JsonInclude.Include.NON_NULL instead of payloadProcessor it's easier to see on bean level.

Reply:
done